### PR TITLE
Streamlined regression tests

### PR DESCRIPTION
**Pull request by mdlpstsci**
These changes address Issue #458.

### CODE REVIEW

File: tests/wfc3/test_uvis_12single.py [hunk 1]
```diff
@@ -27,6 +27,8 @@ def _single_raw_calib(self, rootname):
 
     # Ported from ``calwf3_uv_12``.
     @pytest.mark.parametrize(
+        'rootname', ['iaao09l0q'])
+        """
```
Comment (pllim) [inline tests/wfc3/test_uvis_12single.py]: I'd rather this is commented out than being turned into a docstring. Same comment applies to all other similar occurrences.

File: tests/wfc3/test_uvis_15single.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 from ..helpers import BaseWFC3
 
-@pytest.mark.slow
+@pytest.mark.skip
```
Comment (pllim) [inline tests/wfc3/test_uvis_15single.py]: Maybe add a reason why this is skipped?

File: tests/wfc3/test_uvis_12single.py (hunk 1 above)
Comment (mdlpstsci) [reply tests/wfc3/test_uvis_12single.py]: Understood.  I was saving myself some typing.  I actually want to remove many of these datasets, but I would rather do the actual removal when I have time to make sure I am absolutely not deleting something that is useful.

Comment (mdlpstsci) [reply tests/wfc3/test_uvis_12single.py]: Fixed.

File: tests/wfc3/test_uvis_15single.py (hunk 2 above)
Comment (mdlpstsci) [reply tests/wfc3/test_uvis_15single.py]: Added comment.

File: tests/wfc3/test_uvis_15single.py [hunk 3]
```diff
@@ -3,7 +3,10 @@
 
 from ..helpers import BaseWFC3
 
-@pytest.mark.slow
+# This test is curretnly skipped as it contains 100 (!) files to
```
Comment (pllim) [inline tests/wfc3/test_uvis_15single.py]: I was thinking more like `skip(reason='...')` but this works too. For reference: http://doc.pytest.org/en/latest/skipping.html#skipping-test-functions

```suggestion
# This test is currently skipped as it contains 100 (!) files to 
```

Comment (mdlpstsci) [reply tests/wfc3/test_uvis_15single.py]: Fixed typo and use pytest facilities when appropriate.

**Pull request merged**